### ISSUE TITLE
Either a problem with the 'catch errors' or the 'colors and crayons' libraries

### ISSUE THREAD

**Issue by ToonTalk** [jmoenig/Snap#3133]
![image](https://user-images.githubusercontent.com/2138696/199373850-eed1b9a8-8fa9-4877-a3f5-578a95d0b7b7.png)

What I want to do is check if some text is the name of a color. I could of course scrap together a list of 100s (or is it 1000s) of names.

**Comment by brianharvey** [comment]
I think the problem is that SAFELY TRY doesn't catch user-code-generated errors.

**Comment by joecooldoo** [comment]
> I think the problem is that SAFELY TRY doesn't catch user-code-generated errors.
Correct! I've tried testing this and user-generated errors override the safely try blocks.
